docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -110,6 +110,23 @@ This is what the full working test looks like:
 @@snip [TestKitFragmentSpec.scala](../../../../../test/scala/docs/http/scaladsl/server/TestKitFragmentSpec.scala) { #source-quote }
 
 
+## Accounting for Slow Test Systems
+
+Sometimes, the timeout defined for a route is finely tuned for our production environment, but our Continuous
```

Comment:
Hmm, I'll rephrase it.
I meant, one might have its application correctly configured with the right timeouts for each route, but tests on a slow machine run slower than such timeouts...